### PR TITLE
OBJ-384 bugfix for printable chars comma

### PR DESCRIPTION
**Pull request by mattch1**
Comma should have been "," not ", " added this to the unit test and checked that other charcaters do not have trailing spaces.

### CODE REVIEW

**Review by ICoxCH — APPROVED** [review]

File: test/utils/string.formatter.spec.unit.ts [hunk 1]
```diff
@@ -31,10 +31,10 @@ describe("String formatter tests", () => {
   });
 
   it("should not remove printable chars", () => {
```
Comment (ICoxCH) [inline test/utils/string.formatter.spec.unit.ts:33]: I'd suggest adding a unit test where the preFormattedString is a string of the entire set of acceptable characters, and then check the length of the returned string === length of the preFormattedString.

Comment (markpit) [reply test/utils/string.formatter.spec.unit.ts:33]: i'd agree, we should test all the allowable characters are actually allowed

Comment (mattch1) [reply test/utils/string.formatter.spec.unit.ts]: Added the new test.

**Review by ICoxCH — APPROVED** [review]

File: test/utils/string.formatter.spec.unit.ts [hunk 2]
```diff
@@ -30,11 +29,19 @@ describe("String formatter tests", () => {
...
     const formattedString: string = removeNonPrintableChars(preFormattedString);
 
-    expect(formattedString).toEqual("hello. This string is not dodgy !@£$%&*{}()€#");
+    expect(formattedString).toEqual("hello. This string is not dodgy, !@£$%&*{}()€#");
+  });
+
+  it("should not remove any printable chars when entire set is provided", () => {
+    const preFormattedString: string = "ÀÁÂÄÈÉÊËÌÍÎÏÒÓÔÖÙÚÛÜẀẂŴẄỲÝŶŸàáâäèéêëìíîïòóôöùúûüẁẃŵẅỳýŷÿABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789&@£$€¥.,:;–-‘’'()[]{}<>!“”»«\"?/\\*=#%+ \r\n";
+    const formattedString: string = removeNonPrintableChars(preFormattedString);
+    const test: boolean = preFormattedString === formattedString;
```
Comment (markpit) [inline test/utils/string.formatter.spec.unit.ts:42]: looks like you're testing the same thing twice here. if toEqual isn't enough you can use toStrictEqual

Comment (mattch1) [reply test/utils/string.formatter.spec.unit.ts]: They are different but will change.

**Review by markpit — APPROVED** [review]

**Pull request merged**